scipy/spatial/transform/rotation.py

```diff
@@ -417,5 +518,58 @@ def from_euler(cls, seq, angles, degrees=False):
             raise ValueError("Expected angles to have shape (num_rotations, "
                              "num_axes), got {}.".format(angles.shape))
 
-        quat = _elementary_quat_compose(seq, angles, intrinsic)
+        quat = elementary_quat_compose(seq, angles, intrinsic)
         return cls(quat[0] if is_single else quat, normalized=True)
+
+    def as_euler(self, seq, degrees=False):
+        """Return the Euler angles representation of the Rotation.
+
+        This function returns a numpy.ndarray of shape (N, 3) or (3,) depending
+        on how the object was initialized. The algorithm presented in [2]_ has
+        been adapted for our use to extract Euler angles. The paper presents an
+        algorithm for extracting Euler angles from transformation matrices, as
+        opposed to rotation matrices. Thus, the matrix representation used in
+        the paper is a transpose of the direction cosine matrix representation
+        returned by the `as_dcm` function.
+
```

Comment:
Definitely should write about ranges of the returned angles (we have two cases now).